sql/moz-fx-data-shared-prod/external/chrome_extensions/view.sql

```diff
@@ -27,6 +27,8 @@ SELECT
   extension_updated_date,
   category,
   trader_status,
-  featured
+  featured,
+  verified_domain,
+  IF(verified_domain IS NOT NULL, TRUE, FALSE) AS has_verified_domain
```

Comment:
A shorter version for this would could be:
```
SELECT
  IF("example.com" IS NOT NULL, TRUE, FALSE) AS has_verified_domain,
  "example.com" IS NOT NULL AS has_verified_domain2,
```